walletrpc/service.proto

```diff
@@ -101,6 +108,7 @@ service CompactTxStreamer {
 
     // Return the txids corresponding to the given t-address within the given block range
     rpc GetAddressTxids(TransparentAddressBlockFilter) returns (stream RawTransaction) {}
+    rpc GetAddressBalance(stream Address) returns (Amount) {}
```

Comment:
Where did we land on discussing whether this should be `stream` or `repeated`? Were there any arguments for making this a stream? I think passing an array of addresses, rather than a stream, might be slightly easier on the client side because you can make it a blocking or non-blocking call.

I believe passing an array here would let clients create simple [unary rather than streaming requests](https://grpc.io/docs/what-is-grpc/core-concepts/#client-streaming-rpc).

Reply:
Doing so might involve using a wrapper message such as:

```
message AddressList {
    repeated Address addresses = 1;
}

. . .

 rpc GetAddressBalance(AddressList) returns (Amount) {}
```

which is probably fine (unless there is an easier way).